cypress/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "konto.bratislava.sk",

```

Comment:
problably does not matter, but just in case it'd lead to some weirdness later, give this a different name than the other package.json in this repo